### ISSUE TITLE
Add more Ethereum state DB focused example accessing account values

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/merkle-patricia-tree#35]
Add example from https://github.com/ethereumjs/merkle-patricia-tree/issues/32#issuecomment-363654149 to the README

**Comment by rkernegger** [comment]
This is my suggestion:
```
var level = require('level')
var rlp = require('rlp')
var ethutil = require('ethereumjs-util')

var Trie = require('merkle-patricia-tree/secure')
var Account = require('ethereumjs-account').default
var BN = ethutil.BN

var stateRoot = 'STATE_ROOT_OF_A_BLOCK'

var db = level('YOUR_PATH_TO_THE_GETH_CHAINDATA_FOLDER')
var trie = new Trie(db, stateRoot)

var address = 'AN_ETHEREUM_ACCOUNT_ADDRESS'

trie.get(address, function (err, data) {
  if (err) return cb(err)

  var acc = new Account(data)
  console.log('-------State-------')
  console.log(`nonce: ${new BN(acc.nonce)}`)
  console.log(`balance in wei: ${new BN(acc.balance)}`)
  console.log(`storageRoot: ${ethutil.bufferToHex(acc.stateRoot)}`)
  console.log(`codeHash: ${ethutil.bufferToHex(acc.codeHash)}`)

  var storageTrie = trie.copy()
  storageTrie.root = acc.stateRoot

  console.log('------Storage------')
  var stream = storageTrie.createReadStream()
  stream.on('data', function(data) {
    console.log(`key: ${ethutil.bufferToHex(data.key)}`)
    console.log(`Value: ${ethutil.bufferToHex(rlp.decode(data.value))}`)
  })
  .on('end', function() {
    console.log('Finished reading storage.')
  })
})
```
Questions:
-) I wasn't sure if I should leave/add comments or make it as short as possible? 
-) Should we discuss it here or should I make a PR first?

Adapted and updated from 
[1] https://github.com/ethereumjs/merkle-patricia-tree/issues/32#issuecomment-363654149
[2] https://medium.com/cybermiles/diving-into-ethereums-world-state-c893102030ed
[3] http://wanderer.github.io/ethereum/nodejs/code/2014/08/12/running-contracts-with-vm/

**Comment by holgerd77** [comment]
Hi Rene, great, thanks! 😀 Someone will have a closer look during the next days.

**Comment by holgerd77** [comment]
Hi Rene, like this, I think this should be self-explanatory enough. Do you want to open a PR on this and add as a dedicated example - maybe headed with "Read Account Storage from Geth DB" or similar - to the `README` examples?

That would be great! 😄

**Comment by rkernegger** [comment]
Sure :)

**Comment by holgerd77** [comment]
Closed by #89.
